scripts/qemu.py

```diff
@@ -124,6 +139,8 @@ def _post_launch(self):
     def _post_shutdown(self):
         if not isinstance(self._monitor_address, tuple):
             self._remove_if_exists(self._monitor_address)
+        if not isinstance(self._console_address, tuple):
+            self._remove_if_exists(self._console_address)
```

Comment:
Also I think we should rework the `_console_address`, `_monitor_address` etc and use `self._managed_files = []` approach as this will get messy.